tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/QueryFixerMain.java

```diff
@@ -1,19 +1,196 @@
 package com.google.cloud.bigquery.utils.queryfixer;
 
-import com.google.common.flogger.FluentLogger;
+import com.google.cloud.bigquery.*;
+import com.google.cloud.bigquery.utils.queryfixer.entity.FixOption;
+import com.google.cloud.bigquery.utils.queryfixer.entity.FixResult;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.*;
+
+import java.util.List;
 
 public class QueryFixerMain {
 
-  private static final FluentLogger logger = FluentLogger.forEnclosingClass();
+  private static CommandLine readFlags(String[] args) {
+    Options options = new Options();
 
-  public static void main(String[] args) {
+    Option option =
+        new Option(
+            /*opt=*/ "c",
+            /*long-opt=*/ "credential",
+            /*hasArg=*/ true,
+            /*description=*/ "The credential file (in JSON) of service account connecting to BigQuery. Otherwise, the default application-login credential will be used.");
```

Comment:
nit: of 'the' service account 
Should user provide the path to the json file here?

Reply:
Yes, this flag actually receives the path of the json file, rather than the content.

Reply:
I have modified the description.